blocks/init/src/Blocks/components/card/components/card-editor.js

```diff
@@ -19,8 +17,11 @@ export const CardEditor = (attributes) => {
 		selectorClass = componentClass,
 		blockClass,
 		additionalClass,
+		uniqueWrapperId,
 	} = attributes;
 
+	const unique = uniqueWrapperId ?? useMemo(() => getUnique(), []);
```

Comment:
The same would apply to the JS function, like its PHP counterpart.

Reply:
I have some ideas about this but in the next pr